### ISSUE TITLE
Force certificate rotation after creating the cluster

### ISSUE THREAD

**Issue by cfergeau** [code-ready/snc#27]
We currently need to wait 24h after creating the cluster before getting certificates valid for one month. We should have a way of forcing the rotation of the certificates without waiting for 24h

Experimenting with the commands below seemed to help:
```
# validity is 30 times the base
oc create -n openshift-config configmap unsupported-cert-rotation-config --from-literal='base=30s'

# forcing rotation
oc get secret -A -o json | jq -r '.items[] | select(.metadata.annotations."auth.openshift.io/certificate-not-after" | .!=null and fromdateiso8601<='$( date --date='+1year' +%s )') | "-n \(.metadata.namespace) \(.metadata.name)"' | xargs -n3 oc patch secret -p='{"metadata": {"annotations": {"auth.openshift.io/certificate-not-after": null}}}'

# Wait ~ 5-10 minutes

# Make sure at least the apiserver serving cert has 15 min validity (change your cluster name based on your kubeconfig)
openssl s_client -connect api.tnozicka-1.devcluster.openshift.com:6443 | openssl x509 -noout -dates


# go back to normal certrotation setting
oc delete -n openshift-config configmap unsupported-cert-rotation-config
```

**Comment by praveenkumar** [comment]
I am all up for it if the resulted cluster is healthy, Conversation which I had sometime back with installer folks look like even when initial `oc get co` says required operator are in healthy state then also it need to give some time to reconcile the cluster before putting a workload, I am not sure how quickly that reconcile can happen when we are putting `~30sec` limit to the cert. Looks like we need to test it before make it part of our bundle creation script.

**Comment by anjannath** [comment]
After doing the above steps, how to check if the certificates were rotated?.. i waited around 30mins and checked the certificates in the master.. and found the same 1 day validity
```
[root@crc-l8gfp-master-0 pki]# openssl x509 -in kubelet-client-current.pem -noout -dates 
notBefore=May 28 08:05:00 2019 GMT
notAfter=May 29 07:48:17 2019 GMT 

$ ./oc get csr
NAME AGE REQUESTOR CONDITION
csr-4mp5f 113m system:node:crc-l8gfp-master-0 Approved,Issued
csr-b5jnq 113m system:serviceaccount:openshift-machine-config-operator:node-bootstrapper Approved,Issued 
```

**Comment by anjannath** [comment]
To force cert rotation, we have seen in BZ [1] and over slack channels, only the following command is required, however in my attempts i haven't been able to force the rotation using this.
```
# forcing rotation
oc get secret -A -o json | jq -r '.items[] | select(.metadata.annotations."auth.openshift.io/certificate-not-after" | .!=null and fromdateiso8601<='$( date --date='+1year' +%s )') | "-n \(.metadata.namespace) \(.metadata.name)"' | xargs -n3 oc patch secret -p='{"metadata": {"annotations": {"auth.openshift.io/certificate-not-after": null}}}'
```
https://bugzilla.redhat.com/show_bug.cgi?id=1736800#c9

**Comment by praveenkumar** [comment]
Fixed using #124
